packages/trpc/server/routers/loggedInViewer/deleteCredential.handler.ts

```diff
@@ -206,7 +206,6 @@ export const deleteCredentialHandler = async ({ ctx, input }: DeleteCredentialOp
                   bookingFields: true,
                   seatsPerTimeSlot: true,
                   seatsShowAttendees: true,
-                  seatsShowAvailabilityCount: true,
```

Comment:
This was deleted because we didn't use it?

Reply:
Searched through the file and this wasn't being used anywhere else.

Reply:
Yeah linting can do that